Misc/NEWS.d/next/Library/2023-05-14-11-46-21.gh-issue-87646.DNJjDX.rst

```diff
@@ -0,0 +1,3 @@
+`tempfile.NamedTemporaryFile` and  `tempfile.TemporaryDirectory` includes a
```

Comment:
The docs workflow is failing as it cannot resolve the single ` . What's the right name to link it?

Reply:
resolved in [ee6435b](https://github.com/python/cpython/pull/104466/commits/ee6435b52832cb6a102391bfddc92883170249f9)